source/_components/media_player.monoprice.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: "Monoprice 6-Zone Amplifier"
+description: "Instructions on how to integrate Monoprice 6-Zone Home Audio Controller into Home Assistant."
+date: 2017-10-02 11:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mp-logo.svg
+ha_category: Media Player
+ha_release: 0.25
```

Comment:
Please update.

Reply:
Which release version should I put there? 0.55?